packages/components/src/button/style.scss

```diff
@@ -90,7 +90,7 @@
 			box-shadow:
 				inset 0 -1px 0 color(theme(button) shade(50%)),
 				0 0 0 1px $white,
-				0 0 0 3px $blue-medium-focus;
+				0 0 0 3px color(theme(button) shade(50%));
```

Comment:
Would it be expected that the value produced by this will be different for the _default_ value? In other words, is `color(#0085ba shade(50%))` the same  as what was previously `$blue-medium-focus`? And if it isn't, do we care to consider it a regression?

Reply:
Hmm, good point. Perhaps this should be `color(theme(button))`, or a brighter shade. Based on the blue focus rect – looks closer to regular button blue; `color(theme(button) shade(50%))` is likely too dark. I think this PR needs (more) design feedback!

<img width="659" alt="screen-shot-2019-06-25-at-2 43 28-pm" src="https://user-images.githubusercontent.com/4167300/60136943-adcc2980-97f9-11e9-8820-8a9e6b7a154b.png">

Reply:
My first inclination was to say that this should be `color(theme(button))`, but I dug into it a bit more and realized why our current focus color doesn't match the default blue color already. Looks like the default blue color is `#0085ba`, and `$blue-medium-focus` works out to `#007cba`. These two colors are _really_ close, but the main difference is that the focus color passes WCAG AA contrast tests, whereas the default blue color doesn't. I think that's why our focus color is not currently the same as `color(theme(button))`. 

For this PR, I think we should use `color(theme(button) shade(5%))`. This works out to `#007eb1`, which passes the WCAG AA contrast test, is _really_ close to our current color, and is also [used elsewhere](https://github.com/WordPress/gutenberg/blob/9a1214beee75f04b542448456d901d486e267b2a/packages/components/src/button/style.scss#L80) in the file.

Reply:
Awesome, thanks – will make that change asap :)